src/Sarif.UnitTests/Visitors/RebaseUriVisitorTests.cs

```diff
@@ -191,14 +191,6 @@ public void RebaseUriVisitor_VisitFileData_RebasesAllTheThings()
             string uriRootText = "file:///home/buildAgent/";
             string toolsRootBaseId = "TOOLS_ROOT";
 
-            visitor.FileDataParentKeys.Count.Should().Be(4);
-            visitor.FileDataParentKeys.Where(k => k == null).Count().Should().Be(3);
-            visitor.FileDataParentKeys.Where(k => k != null && k.StartsWith(uriRootText)).Count().Should().Be(1);
-
-            visitor.FileDataKeys.Count.Should().Be(4);
-            visitor.FileDataKeys.Where(k => k != null && k.StartsWith(uriRootText)).Count().Should().Be(3);
-            visitor.FileDataKeys.Where(k => k != null && k.StartsWith("#" + toolsRootBaseId + "#")).Count().Should().Be(1);
```

Comment:
>visitor [](start = 12, length = 7)

This test still fails. Was this change intended to fix it?

Reply:
No, just an incidental change to allow for compilation: there are no more file data keys persisted by this visitor

---
In reply to: [247734541](https://github.com/Microsoft/sarif-sdk/pull/1212#discussion_r247734541) [](ancestors = 247734541)